docs/FAQ.md

```diff
@@ -193,6 +193,13 @@ The plan is that once the design of desktop wallet and browser wallet get stable
 
 We currently have a desktop [wallet](https://github.com/alephium/desktop-wallet/releases/latest) and the team is working on mobile and [web extension wallets](https://github.com/alephium/extension-wallet/releases/latest). We want to offer the possibility to store ALPH on hardware wallets. We are currently pursuing being supported on Ledger. Please note that it is a lengthy process and won't be completed overnight.
 
+### When importing my seed to another desktop wallet, is there a way to import all generated addresses with it?
```

Comment:
Do you guys think it would be better if we have a FAQ under wallet category? I am slightly hesitating on this, but more inclined to do it.

Reply:
It's not that clear. I meant a new FAQ page under this path `https://wiki.alephium.org/wallet/`

Reply:
I would personally find it confusing if every section of the wiki had it's own FAQ. Ithink its better to keep it as it is now, with a section for frequently asked questions about the wallet in the main FAQ. Mainly because i believe it makes it easier to maintain the FAQ.

Reply:
No strong opinion on this for now. Let's re-discuss this when the FAQ page becomes bloated.